jkube-kit/profile/src/main/java/org/eclipse/jkube/kit/profile/ProfileUtil.java

```diff
@@ -100,16 +100,19 @@ private static Profile inheritFromParentProfile(Profile aProfile, File resourceD
      * @param resourceDirs resource directory where to lookup the profile (in addition to a classpath lookup)
      * @return the merged configuration which can be empty if no profile is given
      * @param config the provided configuration
-     * @throws IOException
      */
     public static ProcessorConfig blendProfileWithConfiguration(ProcessorConfigurationExtractor configExtractor,
                                                                 String profile,
                                                                 List<File> resourceDirs,
-                                                                ProcessorConfig config) throws IOException {
-        // Get specified profile or the default profile
-        ProcessorConfig profileConfig = extractProcesssorConfiguration(configExtractor, profile, resourceDirs);
+                                                                ProcessorConfig config) {
+        try {
+            // Get specified profile or the default profile
+            ProcessorConfig profileConfig = extractProcesssorConfiguration(configExtractor, profile, resourceDirs);
 
-        return ProcessorConfig.mergeProcessorConfigs(config, profileConfig);
+            return ProcessorConfig.mergeProcessorConfigs(config, profileConfig);
+        } catch (IOException ioException) {
+            throw JKubeException.launderThrowable("Error in extracting processor configuration", ioException);
+        }
```

Comment:
The point is not to have this. You can safely remove the throws IOException from the extractProcesssorConfiguration (sic) method.

Please, take your time and do things properly and thinking twice. It's the third iteration already for a really simple change.

Reply:
Sorry, I think it should be okay now